src/components/memberDetailPage/memberDetailPage.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { useParams } from "react-router-dom";
+import { useDispatch, useSelector } from 'react-redux';
+
+import { memberActions } from '../../state/actions';
+
+function MemberDetailPage() {
+    const member = useSelector(state => state.member);
+    const { id } = useParams();
+    const dispatch = useDispatch();
+    
+    useEffect(() => {
+        dispatch(memberActions.getMemberById(id));
+    }, [dispatch, id]);
+
+    return (
+        <div className="col-lg-12">
+            <h3>Member Detail:</h3>
+            <ul>
```

Comment:
For ul and li to, material-ui has `<List component="ul">` and `<ListItem>` components that we can use.